src/kernel/src/fs/dev/dmem1.rs

```diff
@@ -24,12 +28,20 @@ impl VFileOps for Dmem1 {
     fn ioctl(
         &self,
         _: &crate::fs::VFile,
-        _: u64,
-        _: &mut [u8],
+        com: u64,
+        mut data: &mut [u8],
         _: &Ucred,
         _: &VThread,
     ) -> Result<(), Box<dyn Errno>> {
-        todo!()
+        match com {
+            Self::COM10 => {
+                data.write_u64::<LittleEndian>(Self::TOTAL_SIZE as u64)
+                    .unwrap();
```

Comment:
Better to write as a native endian instead unless the caller expect little-endian.